### DISCUSSION TITLE
Optional Email field to send the encrypted note

### DISCUSSION THREAD

**Discussion by Super-Gee** [mprimi/portable-secret#22] [Ideas]
I guess it would make sense, once I encrypt a note, to have not only a download link but also an email address field to send it directly as an attachment and another field for a note. This would trigger the PHP server's send mail function.
I would not use it, as PHP + mail send has always been tricky to setup for a no dev like me, but i guess some people would.

**Comment by mprimi** [comment]
You can encrypt a note and send it as email attachment. 

Having the server-side web application send the email for you adds complexity and doesn't really buy you anything in terms of privacy (the opposite in fact).
